src/templates/app/trade/explanation.jsx

```diff
@@ -356,7 +356,7 @@ const Explanation = () => (
             </div>
 
             <div id='note_risefall' className='invisible'>
-                <Note text={it.L('Rise/Fall contracts will be refunded at the purchase price if the following conditions occur: (1) there are less than 2 ticks between the start and end times, or (2) for contracts of tick duration, if the contract doesn\'t end within 5 minutes, or (3) for contracts starting in the future, if there are no ticks in the 5 minutes before the contract\'s start time.')} />
+                <Note text={it.L('Rise/Fall contracts will be refunded if:[_1]• There are less than 2 ticks between the start and end times [_1]• The contract doesn\'t end within 5 minutes (for tick duration contracts)', '<br />')} />
```

Comment:
@mustofa-binary instead of putting it in a br tag can you add separate texts? better not send `•` to translators